App.Stocks/Database/StockDbContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using App.Models.Stocks;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace App.Stocks.Database
+{
+	public class StockDbContext : DbContext
+	{
+		public DbSet<Company> Companies { get; set; } 
+		public DbSet<Stock> Stocks { get; set; }
+		public StockDbContext(DbContextOptions<StockDbContext> options) : base(options) { }
+
+		protected override void OnModelCreating(ModelBuilder builder)
+		{
+
```

Comment:
part of your configuration is placed on data annotations and part is in fluent api. This is not good practice, but fixing it is optional